.github/workflows/end-to-end.yml

```diff
@@ -0,0 +1,40 @@
+name: End to end tests
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # every day at midnight
+
+jobs:
+  test:
+    if: github.repository == 'SubstraFoundation/substra-tests' # Prevent the test from running on forks
+    name: Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        name: Set up python
+        with:
+          python-version: 3.7
+      - uses: azure/setup-helm@v1
+        with:
+            version: 'v3.2.3'
+      - uses: azure/setup-kubectl@v1
+        with:
+            version: 'latest'
```

Comment:
shouldn't we have a fixed version here?

Reply:
The previous script used latest but we can change this. I don't think there is an issue with using latest as it seems quite unlikely that k8s releases a version with a breaking change.